suite-native/module-accounts-import/src/components/AccountImportLoader.tsx

```diff
@@ -1,41 +1,76 @@
-import React from 'react';
+/* eslint-disable global-require */
```

Comment:
Please remove this. I don't see any `require` in this file so there is no need for it.